python/src/trezorlib/transport/webusb.py

```diff
@@ -96,9 +100,10 @@ def write_chunk(self, chunk: bytes) -> None:
                 )
             return
 
-    def read_chunk(self) -> bytes:
+    def read_chunk(self, timeout: float | None = None) -> bytes:
         assert self.handle is not None
         endpoint = 0x80 | self.endpoint
+        start = time.time()
         while True:
             try:
                 chunk = self.handle.interruptRead(
```

Comment:
It seems that `interruptRead` doesn't return `None` on timeout - added a small fix here: https://github.com/trezor/trezor-firmware/pull/4748/commits/a866ffba8fae233f4c12b9e30e28fd6e86291407